packages/request/src/index.ts

```diff
@@ -31,6 +31,10 @@ export interface RequestResult<O = void, E = Error> {
     accept(): void;
 }
 
+export interface RequestResponse<O, E> extends RequestResult<O, E>{
+    request: () => void;
```

Comment:
Still I expect this function to be `refresh` since we already sent a http request

Reply:
done